simpleSequential/reverseFiveIntegerList.py

```diff
@@ -3,3 +3,7 @@
 # Ex:
 #   reverseFiveIntegerList([-2, -1, 0, 1, 2]) # return [2, 1, 0, -1, -2]
 #   reverseFiveIntegerList([3, 5, 9, -2, -7]) # return [-7, -2, 9, 5, 3]
+from typing import List
+
+def reverseFiveIntegerList(lst: List[int]) -> List[int]:
+    return ([lst[4], lst[3], lst[2], lst[1], lst[0]])
```

Comment:
Why do you need the "()"? It makes me confused when reading it. I thought this was a tuple.